app/helpers/evidence_helper.rb

```diff
@@ -0,0 +1,11 @@
+module EvidenceHelper
+  def cancel_path(back_to)
+    if back_to == 'issue'
+      [current_project, @evidence.issue]
+    elsif @evidence.persisted?
+      [current_project, @node, @evidence]
+    else
+      [current_project, @node]
+    end
+  end
+end
```

Comment:
@sean-yeoh is there a way to further improve this so we are not partly duplicating some of this code in the evidence controllers update action?